### DISCUSSION TITLE
Today comparison to be based on the same amount of time

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#344] [Anything Else]
"One request that I would find helpful is having the stat comparisons from the previous day based on the same time the previous day rather than the total. The comparisons almost always show some -X% because you are comparing a 24 hour period yesterday to a 24 - X hour period today."

(request from HN)

**Comment by metmarkosaric** [comment]
The comparison has now been fixed for "This Month" but remains to be fixed for "Today"

**Reply by mmasouras** [reply]
And for "Last Week"?

**Reply by Vigasaurus** [reply]
That'll have to get fixed with Today - as it's not trivial since the internal handling of queries needs to change from just a Date to a DateTime - so they both need hour granularity and thus more effort.

**Comment by mmasouras** [comment]
Is there a tracking ticket for this requirement? I can't find one in the issues list.

**Reply by metmarkosaric** [reply]
hi @mmasouras! the work hasn't started on this yet unfortunately so we don't have any more info other than this thread

**Comment by L1quid** [comment]
As the end user, the easy solution for me would be to simply hide the comparisons for "today".  I know "today" is incomplete data, so I do not find any comparison particularly useful there.

(Does the UI even mention what the previous / comparison period is for the "today" view?  Yesterday?  Same day last week?)

**Comment by metmarkosaric** [comment]
Fixed! We now compare today against the same number of hours yesterday!
